examples/game/lib/layer.dart

```diff
@@ -0,0 +1,19 @@
+part of sprites;
+
+class Layer extends Node with SpritePaint {
+  Paint _cachedPaint = new Paint()
+    ..setFilterQuality(FilterQuality.low)
+    ..isAntiAlias = false;
+
+  void _prePaint(PaintingCanvas canvas, Matrix4 matrix) {
+    super._prePaint(canvas, matrix);
+
+    _updatePaint(_cachedPaint);
+    canvas.saveLayer(null, _cachedPaint);
```

Comment:
null -> Do you not know the bounds of the Node?  saveLayer is more efficient if you can supply the bounds for the offscreen layer.